shipyard_blueprints/salesforce/shipyard_salesforce/salesforce_utils.py

```diff
@@ -0,0 +1,112 @@
+import json
+
+from typing import Optional
+from requests import Response
+from shipyard_templates import Crm, ExitCodeException
+
+
+def handle_request_errors(response: Response) -> None:
```

Comment:
Do you think something like this function could be used across other CRM vendors? Or is this particularly specific to Salesforce?

Reply:
this would be useful for vendors that have a significant number of error statuses